src/gc/transaction_level_gc_manager.cpp

```diff
@@ -53,7 +53,10 @@ void TransactionLevelGCManager::Running(const int &thread_id) {
     auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
     auto max_cid = txn_manager.GetMaxCommittedCid();
 
-    PL_ASSERT(max_cid != MAX_CID);
+    if (max_cid == MAX_CID) {
```

Comment:
please note that the assertion has been commented out. When the system is started but it never processes any transaction, then we may see max_cid == MAX_CID.

Comments added.